Simple patch to add kfreebsd support, created by Cristoph Egger
(cristoph@debian.org).

Index: cfengine3/configure.ac
===================================================================
--- cfengine3.orig/configure.ac	2012-05-31 11:18:15.542107770 -0500
+++ cfengine3/configure.ac	2012-05-31 11:22:27.491090289 -0500
@@ -616,7 +616,7 @@
 	AC_DEFINE(IRIX)
         CFLAGS="$CFLAGS -w"
         ;;
-   linux*)
+   linux*|*bsd*gnu)
 	AC_DEFINE(LINUX, [], [Linux build])
 	AC_CHECK_LIB(nss_nis, yp_get_default_domain)
         AC_CHECK_LIB(acl,main)
